### ISSUE TITLE
MoaiCli no longer building android under Windows 10

### ISSUE THREAD

**Issue by NickoftheSouth** [halfnelson/moaicli#19]
Everything is downloaded correctly (though I had to disable real-time malware protection as it kept quareneting the minGW.exe installer), but the build fails.  
It seems that the paths are being confused on one of the updated tools, or because of windows 10.  

The error looks like this:

` Building With              -DDISABLED_EXT='ADCOLONY;BILLING;CHARTBOOST;CRITTERCISM;FACEBOOK;PUSH;TAPJOY;TWI
      TTER;TSTOREWALL;TSTOREGAMECENTER;PLAYSERVICES'  -DMOAI_BOX2D=1 -DMOAI_CHIPMUNK=1
      -DMOAI_CURL=1 -DMOAI_CRYPTO=1 -DMOAI_EXPAT=1 -DMOAI_FREETYPE=1
      -DMOAI_HTTP_CLIENT=1 -DMOAI_JSON=1 -DMOAI_JPG=1 -DMOAI_LUAEXT=1
      -DMOAI_MONGOOSE=1 -DMOAI_OGG=1 -DMOAI_OPENSSL=1 -DMOAI_SQLITE3=1
      -DMOAI_TINYXML=1 -DMOAI_PNG=1 -DMOAI_SFMT=1 -DMOAI_VORBIS=1 -DMOAI_UNTZ=1
      -DMOAI_LUAJIT=1 -DBUILD_ANDROID=true -DCMAKE_TOOLCHAIN_FILE='C:/Users/Nick
      M/AppData/Local/moaicli/sdks/github.com_moai_moai-dev.git/cmake/host-android/and
      roid.toolchain.cmake'
      -DLIBRARY_OUTPUT_PATH_ROOT='E:/moai_projects/moacli/moai/build/android/android-d
      ebug/bin' -DANDROID_NDK=C:/Users/Nick
      M/AppData/Local/moaicli/deps/android-ndk/android-ndk-r8e -G "MinGW Makefiles"
      -DCMAKE_MAKE_PROGRAM="C:/Users/Nick
      M/AppData/Local/moaicli/deps/android-ndk/android-ndk-r8e/prebuilt/windows/bin/ma
      ke.exe" -DCMAKE_BUILD_TYPE=Debug C:/Users/Nick
      M/AppData/Local/moaicli/sdks/github.com_moai_moai-dev.git/cmake
CMake Error: The source directory "E:/moai_projects/moacli/moai/build/android/android-debug/bin/M/AppData/Local/moaicli/sdks/github.com_moai_moai-dev.git/cmake" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
   Error: Cmake failed to create makefiles  : exitcode false
Exception in thread "main" org.jruby.exceptions.RaiseException: (SystemExit) exit`

It seems is's munged the path of the project with an 'M' and then the path of the sdk in my home directory. Is 'M' some kind of link that no longer works?

**Comment by halfnelson** [comment]
Is your Windows username "Nick M" it is probably just choking on the space
in the username. Which is a bit of a pain .

**Comment by NickoftheSouth** [comment]
Yeah that was the issue, thanks. I made a new user account and it worked.
